AppCenter/AppCenter/Internals/Ingestion/MSIngestionDelegate.h

```diff
@@ -0,0 +1,28 @@
+@protocol MSIngestionProtocol;
+
+@protocol MSIngestionDelegate <NSObject>
+
+@optional
+
+/**
+ * Triggered after the ingestion has suspended its state.
+ *
+ * @param ingestion Ingestion.
+ */
+- (void)ingestionDidSuspend:(__unused id<MSIngestionProtocol>)ingestion;
```

Comment:
I think we should not imply that the parameter won't be used by the delegates here.